### ISSUE TITLE
Faulty core fixes: how to deal with one-line inline control structures ?

### ISSUE THREAD

**Issue by jrfnl** [WordPress-Coding-Standards/WordPress-Coding-Standards#974]
#### Summary:
If an inline control structure (no braces) is a one-liner, it is still a one-liner after the fix, but the closing brace is on the next line (except when the one-liner is followed by a PHP close tag).

Most likely cause: `Generic.ControlStructures.InlineControlStructure` does not differentiate between one-line vs multi-line inline control structures for the fixer.

#### Question:
Should we enforce a new line after the opening brace in these cases ?
(If so, this would probably need to be a new WP native sniff.)

#### Example:

File: `./src/wp-admin/custom-background.php` around line 583

Source:
```php
if ( ! current_user_can('edit_theme_options') || ! isset( $_POST['attachment_id'] ) ) exit;
```

Fixed as:
```php
if ( ! current_user_can( 'edit_theme_options' ) || ! isset( $_POST['attachment_id'] ) ) { exit;
}
```

Expected fix:
```php
if ( ! current_user_can( 'edit_theme_options' ) || ! isset( $_POST['attachment_id'] ) ) {
	exit;
}
```

**Comment by JDGrimes** [comment]
> Should we enforce a new line after the opening brace in these cases ?

Absolutely.

**Comment by jrfnl** [comment]
In that case: how to deal with existing fully inline control structures which currently don't trigger a fixer ?
```php
if ( ! current_user_can('edit_theme_options') || ! isset( $_POST['attachment_id'] ) ) { exit; }
```
and
```html
<input value="<?php if ( isset( $a ) ) { echo $a; } else { echo 'default'; } ?>" />
```

**Comment by JDGrimes** [comment]
Hmmm... good question. The first should probably be fixed to be multiline. The latter should probably really be using a ternary operator instead. Not sure if that can be reasonably auto-fixed though.
